tck/features/expressions/map/Map2.feature

```diff
@@ -28,12 +28,10 @@
 
 #encoding: utf-8
 
-Feature: ExpressionAcceptance
-
-  Background:
-    Given any graph
+Feature: Map2 - Dynamic Value Access
```

Comment:
```suggestion
Feature: Map2 - Dynamic Value Access
# Dynamic value access refers to the bracket-operator – <expression resulting in a map>'['<expression resulting in a string>']' – irrespectively of whether the map key – i.e. <expression resulting in a string> – could be evaluated statically in a given scenario.
```

Please add a comment clarifying the understanding of the term Dynamic value access.